contracts/test/integration/vip/vip15.sol

```diff
@@ -0,0 +1,477 @@
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.MORPHO_COMPOUND_USDC_PCV_DEPOSIT,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause USDC Morpho Compound PCV Deposit"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.MORPHO_COMPOUND_DAI_PCV_DEPOSIT,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause USDC Morpho Compound PCV Deposit"
+            })
+        );
+
+        /// ------- pause psms -------
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.VOLT_USDC_PSM,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause USDC PSM"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.VOLT_DAI_PSM,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause DAI Morpho Compound PCV Deposit"
+            })
+        );
+
+        /// ------- pause allocator -------
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.ERC20ALLOCATOR,
+                arguments: abi.encodeWithSignature("pause()"),
+                description: "Pause ERC20 Allocator"
+            })
+        );
+
+        /// ------- role revoked in core -------
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.CORE,
+                arguments: abi.encodeWithSignature(
+                    "revokePCVController(address)",
+                    MainnetAddresses.MORPHO_COMPOUND_PCV_ROUTER
+                ),
+                description: "Revoke PCV Controller from Morpho Compound PCV Router"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.CORE,
+                arguments: abi.encodeWithSignature(
+                    "revokePCVController(address)",
+                    MainnetAddresses.ERC20ALLOCATOR
+                ),
+                description: "Revoke PCV Controller from ERC20 Allocator"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.CORE,
+                arguments: abi.encodeWithSignature(
+                    "revokeGovernor(address)",
+                    MainnetAddresses.GOVERNOR
+                ),
+                description: "Revoke PCV Controller from ERC20 Allocator"
```

Comment:
```suggestion
                    MainnetAddresses.TIMELOCK_CONTROLLER
                ),
                description: "Revoke GOVERNOR from Timelock"
```

Keep `GOVERNOR` role on Multisig, if needed for whatever cleanup item later. The system won't hold any user funds anyway. Clean it from the Timelock.

Additionally, to completely clean up roles, you should also revoke :
- `GUARDIAN` role from `PCV_GUARDIAN`
- `PCV_GUARD_ADMIN_ROLE` from `PCV_GUARD_ADMIN`
- `PCV_GUARD_ROLE` role from `EOA_1`, `EOA_2`, and `EOA_4`

In the upgraded system, these roles have to be granted with the new CoreV2

Reply:
all roles have been revoked, however, timelock has governor so we can always make an update to the system if needed through proposing via the multisig.